sample-apps/react-native/dogfood/Gemfile

```diff
@@ -3,9 +3,11 @@ source 'https://rubygems.org'
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby '>= 2.6.10'
 
+gem 'cocoapods', '~> 1.12', '>= 1.12.1'
+gem 'fastlane'
+gem 'rubocop', '1.38'
+gem 'rubocop-performance'
+gem 'rubocop-require_tools'
 
-gem "fastlane"
 plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
-eval_gemfile(plugins_path) if File.exist?(plugins_path)
-
-gem 'cocoapods', '~> 1.12', '>= 1.12.1'
```

Comment:
I belive this should not be removed.. cocoapods versions misalignment has caused issues in the past

Reply:
Hey @santhoshvai, actually, it's not removed, I just moved it to the top of the file to keep gems in alphabetic order.